packages/@webex/internal-plugin-metrics/src/new-metrics.ts

```diff
@@ -168,6 +168,61 @@ class Metrics extends WebexPlugin {
     return this.callDiagnosticMetrics.submitClientEvent({name, payload, options});
   }
 
+  /**
+   * Submit a pre-login metric to clientmetrics
+   * @public
+   * @param payload
+   * @param preLoginId - pre-login ID of user
+   * @returns
+   */
+  public postPreLoginMetric(payload: any, preLoginId: string): Promise<any> {
+    // @ts-ignore
+    return this.webex
+      .request({
+        method: 'POST',
+        api: 'metrics',
+        resource: 'clientmetrics-prelogin',
+        headers: {
+          authorization: false,
+          'x-prelogin-userid': preLoginId,
+        },
+        body: payload,
+      })
+      .then((res) => {
+        // @ts-ignore
+        this.webex.logger.log(`Metrics: @postPreLoginMetric. Request successful:`, res);
```

Comment:
is this log not missing the new CALL_DIAGNOSTIC_LOG_IDENTIFIER

Reply:
and should 'Metrics be swapped with NewMetrics so to identify the file where it comes from just like all the others ?

Reply:
and should 'Metrics' be swapped with NewMetrics so to identify the file where it comes from just like all the others ?

Reply:
I've not added CALL_DIAGNOSTIC_LOG_IDENTIFIER because this is not sent within the call-diagnostic file. I've replaced Metrics with NewMetrics.